src/main/scala/uk/gov/nationalarchives/omega/api/services/LocalProducer.scala

```diff
@@ -122,34 +127,36 @@ class LocalProducerImpl(val jmsProducer: JmsProducer[IO], val outputQueue: Queue
     * @return
     */
   def send(replyMessage: String, requestMessage: ValidatedLocalMessage): IO[Unit] =
-    jmsProducer.send { mf =>
-      val msg = mf.makeTextMessage(replyMessage)
-      msg.map { m =>
-        m.setJMSCorrelationId(requestMessage.jmsMessageId)
-        m.setStringProperty(MessageProperties.OMGApplicationID, ApiServiceApp.applicationId)
-        m.setStringProperty(
-          MessageProperties.OMGMessageFormat,
-          jsonContentType
-        )
-        requestMessage.omgMessageTypeId match {
-          case OSLISALS001.entryName =>
-            m.setStringProperty(
-              MessageProperties.OMGMessageTypeID,
-              OutgoingMessageType.AssetLegalStatusSummaryList.entryName
-            )
-          case _ => ()
+    IO.println(s"Reply address: ${requestMessage.omgReplyAddress.value}") *>
```

Comment:
could this removed or replaced with logger?

Reply:
Removed. This was just for debugging.